chia/wallet/nft_wallet/nft_info.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from typing import List
+
+from chia.util.ints import uint64
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class NFTInfo(Streamable):
+    """
+    NFT Info
+    """
+
+    launcher_id: str
+    """
```

Comment:
I appreciate the comments that you've added for each field, but it's hard to read the code in this format. Is it formatted this way to help some IDE with inline code documentation?

Reply:
Python is not my first language, I googled how to add comment in Python. This format is a standard comment style and compatible with document auto-generation tools. Although we don't have plan to generate code document now, but I'd like to keep it in good shape to reduce the future works

Reply:
Can you make these 1-line comments instead of 3-lines then?

e.g.
"""Launcher coin ID"""

Reply:
Here's an example of some source and generated docs.  I don't recall all the details probably and I should reread this but...  https://peps.python.org/pep-0257/

https://github.com/altendky/qtrio/blob/43c7ff24c0be2f7a3df86eef3c6cc5dae2f7ffd3/qtrio/_core.py#L167-L184
https://qtrio.readthedocs.io/en/v0.5.0/core.html#qtrio.Emission

Reply:
Also, if the comment such as `Current NFT coin ID` is more descriptive than the attribute name `nft_coin_id` then we should drop the comment and name it `current_nft_coin_id` I think.

Reply:
Another point about trivial docstrings, if we do get to doc generation or even just checking for docstrings then trivial ones will effectively silence any warnings about missing docstrings.